justfile

```diff
@@ -57,12 +56,16 @@ predict +ARGS: generate-config
         --config-path {{ justfile_directory() }}/config \
         --config-name predict.yaml {{ ARGS }}
 
-# predict with runtime type checking
 predict-debug +ARGS: generate-config
     uv run src/rmind/scripts/predict.py \
         --config-path {{ justfile_directory() }}/config \
         --config-name predict.yaml {{ ARGS }}
 
+@export *ARGS: generate-config
+    uv run src/rmind/scripts/export.py \
+        --config-path {{ justfile_directory() }}/config \
+        --config-name export.yaml {{ ARGS }}
+
```

Comment:
i cant find `src/rmind/scripts/export.py` and corresponding `export.yaml` file

Reply:
ah good catch, this was from the initial experiments; superseded by tests